react/components/Conditions/Separator.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+class Separator extends React.Component {
+  render() {
+    const { label } = this.props
+
+    return (
+      <div>
+        <div
+          style={{
+            marginLeft: -17,
+            width: 'calc(100% + 34px)',
+          }}
+          className="flex flex-row w-100 nowrap items-center mv3">
```

Comment:
Why those flex specificities if it has only one `hr` child?

Reply:
fixed and converted from js to tsx